pkg/sysinfo/sysinfo_linux.go

```diff
@@ -32,6 +33,11 @@ func New(quiet bool) *SysInfo {
 		sysInfo.AppArmor = true
 	}
 
+	// Check if Seccomp is supported.
+	if _, _, err := syscall.RawSyscall(syscall.SYS_PRCTL, syscall.PR_GET_SECCOMP, 0, 0); err != syscall.EINVAL {
```

Comment:
The kill I think will only happen if prctl is itself being filtered, so I wouldnt worry.

You do need the second part of that example, ie `prctl(PR_SET_SECCOMP, SECCOMP_MODE_FILTER, NULL, 0, 0);` checking for `EINVAL` vs `EFAULT` as you are currently only testing for either type 1 or type 2, not type 2.

If after 3.8 is ok, checking for the `Seccomp: ` line in any pid, eg self is maybe ok, it might be that it is set with seccomp type 1 only (you can test by disabling syscall filtering but not original seccomp). I think the first test is better.